includes/shortcodes.php

```diff
@@ -288,6 +295,105 @@ function give_receipt_shortcode( $atts ) {
 		return $login_form;
 	}
 
+    /*
+	 * Collect Data to generate donation receipt args.
+	 *
+	 * @since 1.8.8
+	 */
+    $meta         = give_get_payment_meta( $payment_id );
+    $donation     = give_get_payment_form_title( $meta );
+    $user         = give_get_payment_meta_user_info( $payment_id );
+    $email        = give_get_payment_user_email( $payment_id );
+
+    $donor_name = $user['first_name'].' '.$user['last_name'];
+    $donation_gateway = give_get_gateway_checkout_label( $session['post_data']['give-gateway'] );
+
+    $give_fee_donation_amount = give_format_amount( get_post_meta( $payment_id, '_give_fee_donation_amount', true ) );
+    $give_fee_amount          = give_format_amount( get_post_meta( $payment_id, '_give_fee_amount', true ) );
+    $total_donation = give_payment_amount( $payment_id );
+
+    /*
+	 * Generate Donation Receipt Arguments.
+	 *
+	 * Added donation receipt array to global variable $give_receipt_args to manage it from single variable
+     *
+     * @since 1.8.8
+	 */
+    $give_receipt_args['donation_receipt'][] = array(
+        'name'      => __('Donor', 'give'),
+        'value'     => $donor_name,
+        'display'   => $atts['donor']
+    );
+
+    $give_receipt_args['donation_receipt'][] = array(
+        'name'      => __('Date', 'give'),
+        'value'     => date_i18n( give_date_format(), strtotime( $meta['date'] ) ),
+        'display'   => $atts['date']
+    );
+
+    $give_receipt_args['donation_receipt'][] = array(
+        'name'      => __('Donation Fee', 'give'),
+        'value'     => give_currency_filter( $give_fee_amount, give_get_payment_currency_code( $payment_id ) ),
+        'display'   => false
+    );
+
+    $give_receipt_args['donation_receipt'][] = array(
+        'name'      => __('Donation Amount', 'give'),
+        'value'     => give_currency_filter( $give_fee_donation_amount, give_get_payment_currency_code( $payment_id ) ),
+        'display'   => false
+    );
+
+    $give_receipt_args['donation_receipt'][] = array(
+        'name'      => __('Total Donation', 'give'),
+        'value'     => $total_donation,
+        'display'   => true
+    );
+
+    $give_receipt_args['donation_receipt'][] = array(
+        'name'      => __('Donation', 'give'),
+        'value'     => $donation,
+        'display'   => true
+    );
+
+    $give_receipt_args['donation_receipt'][] = array(
+        'name'      => __('Donation Status', 'give'),
+        'value'     => esc_attr( $status ),
```

Comment:
@mehul0810 I did not see that where is `$status` defined in this function. can you check it?